JavaScript/Topics/13.DOM_in_javascript/README.md

```diff
@@ -0,0 +1,155 @@
+
+
+Below image shows a basic heirarchy in DOM :
+
+
+![DOM ](https://searchengineland.com/figz/wp-content/seloads/2015/05/Microsoft-PowerPoint.jpg)
+
+
+## Use of DOM in JavaScript
+<br>
+With the DOM , javascript get all the power it needs to create dynamic webpage . With DOM :
+
+* JavaScript can change all the HTML elements in the page
+
+* JavaScript can change all the HTML attributes in the page
+
+* JavaScript can change all the CSS styles in the page
+
+* JavaScript can remove existing HTML elements and attributes 
+
+* JavaScript can add new HTML elements and attributes
+
+* JavaScript can react to all existing HTML events in the page
+
+* JavaScript can create new HTML events in the page
+
+
+## Document object in JavaScript
+<br>
+
+A web page or say HTML document can be imagined as a nested set of boxes. Tags such as head and body enclose other tags, which in turn contain other tags or text. For example a simple webpage can have following structure :
+
+![Document]( https://eloquentjavascript.net/img/html-boxes.svg ) 
+
+In the HTML DOM (Document Object Model), everything is a node:
+
+* The document itself is a document node
+* All HTML elements are element nodes
+* All HTML attributes are attribute nodes
+* Text inside HTML elements are text nodes
+* Comments are comment nodes.
+
+The HTML DOM document object is the owner of all other nodes in a  web page. It provides properties and methods to  access all node objects, from within JavaScript.
+
+Now let's talk about few DOM properties .
+
+### HTML | DOM head Property
+
+<br>
+The head property returns the < head> element of the current document and in case if if there are more than one < head> element in the document, this property returns the first one.
+
+E.g. 
+``` 
+var x= document.head ; // Assigns head element to x
+
+console.log(x.innerHTML); // Prints text inside head element 
+
+```
+
+### HTML | DOM body Property
+<br>
+It  returns the content present in the < body> tag. This property is used to view or change the  content present inside the < body> element and sets them with the new specified content.
+
+E.g. 
+```
+ var x = document.body; // Assigns body element to x 
+
+console.log(x.innerHTML); // Print text inside body element 
+
+//New content inside body can also be added 
+ x.innerHTML = "<h1>"
+                + "New Heading added "
+                + "</h1>" + "<br>" + "<h2>"
+                + "New Content Added"
+                + "</h2>";
+```
+
+### HTML | DOM forms Collection
+<br>
+The forms collection returns a collection of all < form> elements present in the document . Since there may be multiple forms in a page , the elements are sorted as they are present in the HTML source code . 
```

Comment:
```suggestion
The forms collection returns a collection of all `<form>` elements present in the document. Since there may be multiple forms on a page, the elements are sorted as they are present in the HTML source code. 
```